src/main/java/com/jalasoft/webservice/model/ImageConvert.java

```diff
@@ -20,7 +21,7 @@
 public class ImageConvert implements IConvert {
 
     @Override
-    public BaseFile Convert(BaseFile model) {
+    public Response Convert(BaseFile model) {
```

Comment:
I think the model class should return a model class or primitive, the Response object should be used in Controller package. what do you think?